remote_monitoring/remote_monitoring/metrics_export.py

```diff
@@ -32,14 +32,13 @@ def get_data(qibocal_output_folder: Path) -> QpuData:
     acquisition_time = report_meta.start_time
     qpu_data = collections.defaultdict(dict)
 
-    for task_id, result in out.history.items():
-        task_id = task_id.id
-        metric = METRICS[task_id]
-        metric_values = getattr(result.results, metric)
-        for qubit_id, qubit_metric in metric_values.items():
-            if isinstance(qubit_metric, list):
-                qubit_metric = qubit_metric[0]
-            qpu_data[qubit_id][metric] = qubit_metric
+    calibration = Calibration.model_validate_json(
+        (qibocal_output_folder / "new_platform" / "calibration.json").read_text()
```

Comment:
```suggestion
        (qibocal_output_folder / "new_platform" /  CALIBRATION ).read_text()
```
In this way if we change the name of the json file it should automatically get updated.